libs/frontend/util-command/src/lib/update-ui/update-ui.ts

```diff
@@ -0,0 +1,22 @@
+import { IUpdateUiData, UiAction } from '@pubstudio/shared/type-command-data'
```

Comment:
In order to handle different update ui actions while supporting redo & undo, I found myself introducing this nested apply function pattern to the app. I wonder if there's a better solution 🤔?

Reply:
The types are a little odd, but I think it's fine!